### DISCUSSION TITLE
Watch Progress

### DISCUSSION THREAD

**Discussion by DANW999** [arabcoders/watchstate#350] [General] [answered]
May I ask if this stores the actual watch progress? I have two Plex servers and I am wondering if I start watching a movie on one of my Plex servers but do not finish it, can I then continue watching that same movie on another Plex server after running the watchstate script?

**Comment by arabcoders** [comment]
Hello 

Unfortunately, due to design choices we aren't able to provide watch state progress, only watched/unwatched. It's something we are looking for a way to implement but due to our initial design choice it's proven to be a bit too hard, i may revisit it at some point in the future.

Thank you.

**Reply by arabcoders** [reply, marked as answer]
@DANW999 As i had free time on my hand i have somewhat working proof of concept for the feature, sadly it's only works with webhooks. So if you are up for it check out the new `state:progress` command and the README.md doc about it.

**Reply by DANW999** [reply]
Thank you @ArabCoders I will be sure to check this out!
